### ISSUE TITLE
[BUG] Use FQDN for device name

### ISSUE THREAD

**Issue by MadScientist88** [SCCM-Zone/CM-SRS-Dashboards#11]
The report is currently only displaying the hostname of the device.
In environments with multiple domains, there can be servers with same hostname in different domains, that makes it hard distinguishing them in the reports.
The view v_RA_System_ResourceNames contains the FQDN of the device.

It would be nice if it per default shows the FQDN or giving the option to display it.

**Comment by Ioan-Popovici** [comment]
It’s using FQDN:

```SQL
, Device            = IIF(Systems.Full_Domain_Name0 IS NOT NULL, Systems.Name0 + '.' + Systems.Full_Domain_Name0, Systems.Name0)
FROM fn_rbac_R_System(@UserSIDs) AS Systems
```
Run the following query in your SSMS and see what you get

```SQL
SELECT 
   DomainName = Systems.Full_Domain_Name0
   , NetBiosName = Systems.Name0
   , FQDN = Systems.Name0 + '.' + Systems.Full_Domain_Name0
FROM fn_rbac_R_System('disabled') AS Systems
```

**Comment by MadScientist88** [comment]
I ran the query, and for around 1800+ servers only around 50 had something in DomainName and FQDN, on the rest it's just NULL.
The only place I found the FQDN to be mentioned every time is in the ResourceNames property in SCCM.

**Comment by Ioan-Popovici** [comment]
I have a feeling that you configured the discovery method to not gather the domain name.
I will look into the ResourceNames table and get back to you tomorrow

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by MadScientist88** [comment]
The only discovery method we have enabled is heartbeat. We can't really use the other once.

**Comment by Ioan-Popovici** [comment]
That explains it. I’ll look into the workaround 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Ioan-Popovici** [comment]
fixed in 4890ec34a36971c025bf518c87fbde8cd137845b
